python/tidy/servo_tidy/tidy.py

```diff
@@ -788,6 +821,17 @@ def get_file_list(directory, only_changed_files=False, exclude_dirs=[]):
 
 
 def scan(only_changed_files=False, progress=True):
+    # check if config file exists
+    if not os.path.exists(CONFIG_FILE_PATH):
+        print("%s config file is required but was not found" % CONFIG_FILE_PATH)
+        sys.exit(1)
+    # load configs from ./servo-tidy.toml
+    with open(CONFIG_FILE_PATH) as content:
+        conf_file = content.read()
+    # check config file for errors
+    config_errors = check_config_file(conf_file, progress)
+    # parse config file
+    parse_config(conf_file)
```

Comment:
To be precise, I'm totally cool with the last commit (with the nits addressed), but with the only change that we don't have to check for `CONFIG_FILE_PATH` before parsing.